manager/controllers/app/m4dapplication_controller.go

```diff
@@ -60,6 +60,10 @@ func (r *M4DApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		log.V(0).Info("The reconciled object was not found")
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
+
+	// Call validate update function from webhook (only if webhooks are disabled - check flag )
+	// ValidateUpdate()
+
```

Comment:
Is this the right location in the reconcile to check?
We should be checking the taxonomy only if there a change in the spec part.
I.e., if we reconcile on error and we already checked the taxonomy once, no need to repeat.